### PR TITLE
WIP Docs

### PR DESCRIPTION
**Pull request by aecay**

### CODE REVIEW

File: docs/design-deployment.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+   }
+
+
+Useful topologies
+=================
+
+While itʼs true that increasing the heft of a test scenario increases
+the load on all miteʼs components, nowhere is the increase more apparent
+than on the runners, the component directly responsible for injecting
```
[the remark this answers is not in the document]
Comment (aecay) [reply docs/design-deployment.rst]: Well, we routinely run with 64 (M25) or 40 (core platform) runners, and only 5 (M25) or 15 (core platform) stats.  That's what I'm aiming to say: that the runner is always the biggest component (by number of replicas)

File: docs/config.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+``context`` argument.  This implements a generic key-value storage
+system for managing journey configuration.  Journey functions can call
+``context.config.get("key")`` to retrieve the value set for a given key.
+
+Managing configuration in this way, rather than as values in python
+files, has several advantages:
+
+- It is easy to parameterize tests for different environments
+- Sensitive data (IP addresses, passwords) can be stored separately from
+  test journey code and encrypted (we use `git-crypt`_ for this purpose)
```
[the remark this answers is not in the document]
Comment (aecay) [reply docs/config.rst]: f60daef

File: docs/design-deployment.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+exporter, and (if warranted) collector and recorder.  The other three have
+8GB of memory and 16 cores each, and each host 16 runner processes.  Our
+system under test consists of a 1:1 replica of the production environment,
+deployed with each weekly release candidate.  The underlying hardware is
+in a corporate datacentre (though it could just as easily correspond to
+servers rented from a colocation facility or VM provider).  We regard
+this as fairly typical of a traditional NFT setup in a medium to large
+tech company.
+
+We have used this infrastructure to inject load of up to 10k tps into our
```
[the remark this answers is not in the document]
Comment (aecay) [reply docs/design-deployment.rst]: 6c019fb

File: docs/design-deployment.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jb098) [reply docs/design-deployment.rst]: I suppose you're correct, I always thought we never even touched the runners but maybe I've been warped by my recent dealings with stats instance increases. I'll shut up.

**Pull request merged**